assets/src/edit-story/components/moveable/index.js

```diff
@@ -27,13 +26,22 @@ import InOverlay from '../overlay';
 
 const DEFAULT_Z_INDEX = 10;
 
+const Moveable = lazy(() =>
+  import(
+    /* webpackPrefetch: true */ /* webpackChunkName: "chunk-react-moveable" */ 'react-moveable'
```

Comment:
Nit: this could be combined to `/* webpackPrefetch: true, webpackChunkName: "chunk-react-moveable" */`